content/help/faq/distributed-tracing/disabling-tracing.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: How can I prevent Istio from sending trace spans to ZipKin?
-weight: 100
+title: How can I prevent Istio from sending trace spans to a backend system?
```

Comment:
usually FAQ are questions people actually ask. Is this how someone asks this question? Usually we are asked...

* How do I disable tracing?
* Is there a kill switch for tracing?

Reply:
Good point. Updated.